app/main.py

```diff
@@ -1 +1,56 @@
-#  write your code here
+from __future__ import annotations
+
+
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            food_points = self.appetite
+            self.is_hungry = False
+            self.appetite = 0
+            return food_points
+        else:
+            self.appetite = 0
```

Comment:
> feed - should print Eating {appetite} food points..., set is_hungry to False and return number of eaten food points if animal is hungry. Otherwise, it should return 0 and print nothing.

No info to set appetite to 0, just return 0